src/eip4824.ts

```diff
@@ -34,29 +34,61 @@ router.get('/:space', async (req, res) => {
 });
 
 router.get('/:space/members', async (req, res) => {
-  let space: any = {};
-
   try {
-    space = await getSpace(req.params.space);
+    const spaceId = req.params.space;
+    const cursor = req.query.cursor || null;
+    const pageSize = 500; // Default page size
+
+    const space = await getSpace(spaceId);
+    if (!space.verified) {
+      return res.status(400).json({
+        error: 'INVALID_SPACE',
+        message: 'The specified space is not verified.'
+      });
+    }
 
-    if (!space.verified) return res.status(400).json({ error: 'INVALID' });
-  } catch (e) {
-    return res.status(404).json({ error: 'NOT_FOUND' });
-  }
+    // Includes space.members, space.admins, space.moderators and voters
+    const combinedMembersResult = await getCombinedMembersAndVoters(
+      spaceId,
+      cursor,
+      pageSize
+    );
 
-  const members = [...space.admins, ...space.moderators, ...space.members].map(
```

Comment:
here we already have `admins` , `moderators` and `members` , is there any reason to get them again in `getCombinedMembersAndVoters` ? If there is no big reason better to read them from here to reduce some load on DB

Reply:
`getCombinedMembersAndVoters` Function is designed to get all the members of the DAO, including Admins, Moderators, Members and Voters. 
 
 To get a unique, consolidated list of all member groups, including pagination we are enforcing this on DB query.
 
 In my previous approach, this was handled at the API level.
  Here's a reference to my old approach,
 https://github.com/snapshot-labs/snapshot-hub/pull/812/commits/3e5bf86cadec031b9cc6cad7240676a05a5a748d

Reply:
> To get a unique, consolidated list of all member groups, including pagination we are enforcing this on DB query.

In that case we can exclude `admins`, `moderators` and `members` from voters within the SQL query,

Reply:
Changes updated.